lib/ansible/modules/cloud/azure/azure_rm_webapp.py

```diff
@@ -504,7 +504,7 @@ def __init__(self):
             id=None,
         )
         self.state = None
-        self.to_do = Actions.NoAction
+        self.to_do = []
```

Comment:
So now "NoAction" and "Delete" constants defined above are not used at all. So I think they should be deleted.
Or maybe just use Boolean variables?